File: kata-5kyu/rot13.js
function rot13(message){
  const code = 'nopqrstuvwxyzabcdefghijklm';
  console.log(message);
  return message.split('')
                .map(el=> code.includes(el) 
                            ? code[el.charCodeAt(0) - 97] 
                            : code.toUpperCase()[el.charCodeAt(0) - 65] || el)
                .join('');
}

File: python-kata-5kyu/rot13.py
def rot13(message):
  abc =  'abcdefghijklmnopqrstuvwxyz'
  code = 'nopqrstuvwxyzabcdefghijklm'
  abc+= abc.upper()
  code+= code.upper()
  result = ''
  for x in message:
    if x in abc:
        result+= code[abc.index(x)]
    else:
        result+= x
  return result 
